tests/unit_tests/main_test.cpp

```diff
@@ -151,14 +153,6 @@ int main(int argc, char** argv) {
 #endif
     }
 
-#if defined(ENABLE_MKLCPU_BACKEND) || defined(ENABLE_NETLIB_BACKEND) || \
```

Comment:
Could you please comment why we don't need this block anymore?

Reply:
This block always adds a CPU device for testing, but CPUs devices are added also in this [block of code](https://github.com/oneapi-src/oneMKL/blob/develop/tests/unit_tests/main_test.cpp#L113-L143). I was going to fix it only for portBLAS but it seems to me that it repeats the logic above so we don't need it.

Reply:
Got it, thank you.